.github/workflows/prepare_release.yml

```
@@ -0,0 +1,65 @@
+---
+
+name: Prepare Release
+
+on:
+  workflow_call:
+    inputs:
+      version:
+        description: 'Module version to be released.'
+        required: true
+        type: string
+      working-directory:
+        description: The working directory where all jobs should be executed. Used for modules in subdirectories like a monorepo or a control repository.
+        default: '.'
+        required: false
+        type: string
+      base-branch:
+        description: 'The branch that will be used as the origin for the release branch.'
+        required: false
+        default: 'master'
+        type: string
+
+env:
+  BUNDLE_WITHOUT: development:test:system_tests
+  GIT_AUTHOR_NAME: Release Automation
+  GIT_AUTHOR_EMAIL: "${{ github.repository_owner }}@users.noreply.github.com"
```

Tying into https://github.com/voxpupuli/gha-puppet/pull/66/files#r1892949324 I think this is needed
```suggestion
  BLACKSMITH_FULL_VERSION: "${{ inputs.version }}"
```